include/linker/common-rom.ld

```diff
@@ -160,3 +160,10 @@
 		KEEP(*(SORT_BY_NAME("._cfb_font.*")))
```

Comment:
This file is generic and not related to ctf, split them out into their own commit.